desktop/package.json

```diff
@@ -13,21 +13,28 @@
 		"email": "wp-desktop@automattic.com"
 	},
 	"homepage": "https://desktop.wordpress.com/",
-	"main": "build/desktop.js",
+	"main": "./dist/index.js",
 	"keywords": [
 		"desktop",
 		"wordpress",
 		"wordpress.com"
 	],
 	"scripts": {
-		"build": "npx electron-builder ${ELECTRON_BUILDER_ARGS} build --publish never",
-		"ci:build-mac": "node bin/build-mac-ci",
+		"dev": "cross-env NODE_ENV=development DEBUG=1 WP_DESKTOP_DEBUG=1 electron app/index.js",
+		"dev:localhost": "cross-env WP_DESKTOP_DEBUG_LOCALHOST=1 yarn run dev",
+		"ci:build-mac": "yarn run build:main && node bin/build-mac-ci",
+		"build": "yarn run build:main && ELECTRON_BUILDER_ARGS=${ELECTRON_BUILDER_ARGS} yarn run build:app",
+		"build:app": "npx electron-builder ${ELECTRON_BUILDER_ARGS} build --publish never",
```

Comment:
Why use `npx` to download and run `electron-builder`, instead of having it as a devDependency and just use `yarn electron-builder`?

(I know this was already there, it is something that can be solved later in a future PR)

Reply:
Done.